### PR TITLE
Small fixes on the ConversationIcon design

### PR DESCRIPTION
**Pull request by skjnldsv**
- Fix alignment and proper sizing of the elements
- Reduced star size for better antialiasing

| Before | After |
|:---------:|:------:|
|Kazam_screenshot_00001|Kazam_screenshot_00000|

### CODE REVIEW

**Comment by nickvergessen** [comment]
`3. to review` or `Draft PR`, you can not have both!?

File: src/components/ConversationIcon.vue [hunk 1]
```diff
@@ -90,41 +91,45 @@ export default {
...
-		background-size: 44px;
-	}
-	&.icon-public,
-	&.icon-contacts,
-	&.icon-password,
-	&.icon-file,
-	&.icon-mail {
-		background-size: 20px;
+	&__avatar {
+		// we request 40px avatars, but
```
Comment (nickvergessen) [inline src/components/ConversationIcon.vue]: we can also make everything 40 instead. I think this was not an intentional mismatch

Comment (skjnldsv) [reply src/components/ConversationIcon.vue]: Well, both are good to me, I really have no preference, but you need to choose :)

**Comment by skjnldsv** [comment]
> `3. to review` or `Draft PR`, you can not have both!?

Wrong button, my bad

File: src/components/ConversationIcon.vue (hunk 1 above)
Comment (nickvergessen) [reply src/components/ConversationIcon.vue]: Let's go for 40 then

Comment (skjnldsv) [reply src/components/ConversationIcon.vue]: Well, minimum clickeable area on nextcloud is 44px. SO I'd rather have everything 44.
What do you prefer @jancborchardt ?

Comment (skjnldsv) [reply src/components/ConversationIcon.vue]: But visually it looks cleaner with 40px

| 40 | 44 |
|:---------:|:------:|
|dev skjnldsv com_call_s934fu82|dev skjnldsv com_call_s934fu82 (1)|

Comment (ma12-co) [reply src/components/ConversationIcon.vue]: Maybe a 44 by 44 clickable square wrapping a 40px avatar?

Comment (skjnldsv) [reply src/components/ConversationIcon.vue]: The avatar component doesn't support it, let's not over complicate things ^^

Comment (ma12-co) [reply src/components/ConversationIcon.vue]: Then I'd also prefer 44px!

**Pull request merged**